charts/alfresco-process-services/values.yaml

```diff
@@ -0,0 +1,185 @@
+    IDENTITY_SERVICE_AUTH: "http://localhost:8080/auth"
+    # -- Alfresco Identity Service force using SSL
+    IDENTITY_SERVICE_SSL_REQUIRED: "none"
+    # -- Alfresco Identity Service resource name to use
+    IDENTITY_SERVICE_RESOURCE: "alfresco"
+    # -- Alfresco Identity Service Attribute to map to user login
+    IDENTITY_SERVICE_PRINCIPAL_ATTRIBUTE: "email"
+    # -- Alfresco Identity Service refresh service token
+    IDENTITY_SERVICE_ALWAYS_REFRESH_TOKEN: "true"
+    # -- Alfresco Identity Service Bearer only toggle
+    IDENTITY_SERVICE_AUTODETECT_BEARER_ONLY: "true"
+    # -- Alfresco Identity Service token storage configuration
+    IDENTITY_SERVICE_TOKEN_STORE: "session"
+    # -- Alfresco Identity Service allow basic authentication
+    # (should only be used over SSL)
+    IDENTITY_SERVICE_ENABLE_BASIC_AUTH: "true"
+    # -- Alfresco Identity Service public client toggle
+    IDENTITY_SERVICE_PUBLIC_CLIENT: "true"
+    # -- Alfresco Identity Service browser initiated logout toggle
+    IDENTITY_SERVICE_USE_BROWSER_BASED_LOGOUT: "true"
+    # -- Alfresco Identity Service application secret
+    IDENTITY_CREDENTIALS_SECRET: ""
+  volumes: []
+  volumeMounts: []
+
+license:
+  secretName:
+
+adminApp:
+  replicacount: 1
+  podSecurityContext: {}
+  image:
+    repository: quay.io/alfresco/alfresco-process-services-admin
+    tag: 24.1.0
+    pullPolicy: IfNotPresent
+    internalPort: 8080
+  service:
+    name: aps-admin
+    type: ClusterIP
+    externalPort: 80
+  livenessProbe: &adminProbe
+    path: /activiti-admin/
+    initialDelaySeconds: 25
+    periodSeconds: 10
+    failureThreshold: 5
+    timeoutSeconds: 5
+  readinessProbe: *adminProbe
+  ingress:
+    path: /activiti-admin
+    maxUploadSize: "5G"
+  environment:
+    # -- Set the JDBC driver Class
+    ACTIVITI_ADMIN_DATASOURCE_DRIVER: "org.postgresql.Driver"
+    # -- Hibernate dialect (must match the driver)
+    ACTIVITI_ADMIN_HIBERNATE_DIALECT: "org.hibernate.dialect.PostgreSQLDialect"
+    # -- activiti-app address
+    ACTIVITI_ADMIN_REST_APP_HOST: "http://localhost"
+    # -- activiti-app port
+    ACTIVITI_ADMIN_REST_APP_PORT: "80"
+    # -- activiti-app username
+    ACTIVITI_ADMIN_REST_APP_USERNAME: "admin@app.activiti.com"
+    # -- activiti-app password
+    ACTIVITI_ADMIN_REST_APP_PASSWORD: "admin"
+  volumes: []
+  volumeMounts: []
+
+database:
+  # -- JDBC url to connect to the external DB
+  url:
+    aps: null
+    admin: null
+  # -- JDBC username to use to connect to the DB
+  username: null
+  # -- JDBC password to use to connect to the DB
+  password: null
+  existingConfigMap:
+    name: null
+    keys:
+      # -- configmap key where to find the URL of the database
+      url: DATABASE_URL
+      # -- configmap key where to find the JDBC driver class to use.
+      # The configmap may leverage the alfresco-repository.db.cm named
+      # template to auto-generate it from the sole url parameter.
+      driver: DATABASE_DRIVER
```

Comment:
look like this is not used and instead we're using the `cm` projected in deployment with `envFrom`